src/components/services/stripe.service.ts

```diff
@@ -63,6 +65,7 @@ async function getAllPrices(isEur?: boolean) {
               priceId: productValue.id,
               storage: storage,
               price: Math.abs(productValue.amount / 100).toFixed(2),
```

Comment:
I know this is not from this PR, but formatting prices in the frontend like this, is going to bring troubles, could be possible that the user sees a pricing in the UI, and another one in the checkout